data/dead_metaphor.json

```diff
@@ -0,0 +1,144 @@
+{
+  "name": "Dead Metaphor",
+  "tagline": "Scottish Breakfast Stout",
+  "first_brewed": "09/2013",
+  "description": "Scottish chocolate breakfast Stout - brewed with Brazilian Sertão coffee, Venezuelan cacao and (for the Scottish element) oatmeal. Smooth, chocolately and rich, with a honeycomb mouthfeel and a rich red berry coffee character. Brewed with our beer blogger friends from The Beer Cast and Hopzine.",
+  "abv": 6.8,
+  "ibu": 30,
+  "target_fg": 1016,
+  "target_og": 1068,
+  "ebc": 86,
+  "srm": 43,
+  "ph": 4.4,
+  "attenuation_level": 76.5,
+  "volume": {
+    "value": 20,
+    "unit": "liters"
+  },
+  "boil_volume": {
+    "value": 25,
+    "unit": "liters"
+  },
+  "method": {
+    "mash_temp": {
+      "temp": {
+        "value": 64,
+        "unit": "celsius"
+      },
+      "duration": 75
+    },
+    "fermentation": {
+      "temp": {
+        "value": 19,
+        "unit": "celsius"
+      }
+    },
+    "twist": {
```

Comment:
Would you mind flattening this to a string I have just been doing it like this
```
"twist": "Cacao: 37.5g at end, Lactose: 125g at end, Ground Coffee: 6.3g at end"
```

The reason being is that as you may find, alot of recipes with twists don't follow a pattern so I figure a string is the safest option